models/user.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var bcrypt = require('bcrypt-nodejs')
+  , totp = require('../lib/totp')
+  , mongoose = require('mongoose')
+  , base32 = require('thirty-two');
+
+var schema = new mongoose.Schema({
+  username: { type: String, unique: true, required: true, dropDups: true },
+  totp_secret: String,
```

Comment:
Is there any reason why we're using _snake_case_?

Please take a look at:

* http://mongoosejs.com/docs/schematypes.html
* http://mongoosejs.com/docs/models.html

Seems like the preferred style is _camelCase_. What do you think?